.env.test.example

```diff
@@ -0,0 +1,7 @@
+DB_CONNECTION_URI=postgres://infisical:infisical@localhost:5430/infisical-test
```

Comment:
Perhaps add a comment in the top of the file that explains what this env file is actually used for.

Reply:
I don't see the purpose of it, because clearly the above file name is a standard practice and says its an env for test. Putting a comment will only make it redundant.

Reply:
Gotcha. I just feel like it's better to have too much than too little documentation/explanation.